src/current/_includes/cockroachcloud/backups/terraform-managed-backups.md

```diff
@@ -0,0 +1,9 @@
+Set the following for the `backup_config` attribute:
+
+- `enabled` controls whether managed backups are enabled or disabled. If you modify the `retention_days` setting even when managed backups are disabled, this will use the one possible change for `retention_days`. Possible values for the `enabled` setting are: `true` or `false`.
```

Comment:
if by "one possible change" you are referring to the other callout where we say you can modify the retention policy only once, is it possible to make this a link to that info?

i see it's in an include but perhaps it could be done with an anchor tag just above the include

if it's too hard because each include is moving around i totally get it

however i think "the one possible change" as standalone text may be too subtle, i'm not a careful enough reader of docs when I use a thing, I only noticed because i just read the other thing

Reply:
Yes, so I linked the `one possible change` to the general section on Retention at the top of the page, because this details the "one possible change".